metaCommand.h

```diff
@@ -29,6 +29,10 @@
 #include <list>
 #include <map>
 
+#ifndef METAIO_USE_NAMESPACE
```

Comment:
This file was probably taken from MetaIO, https://github.com/Kitware/MetaIO/blob/master/src/metaCommand.h

Is this file needed here? Maybe look at its history to find some answers.